phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -3490,7 +3496,11 @@ private void invalidateServerMetadataCache(List<InvalidateServerMetadataCacheReq
             // This will incur an extra RPC to the master. This RPC is required since we want to
             // get current list of regionservers.
             Collection<ServerName> serverNames = admin.getRegionServers(true);
+            PhoenixStopWatch stopWatch = new PhoenixStopWatch().start();
             invalidateServerMetadataCacheWithRetries(admin, serverNames, requests, false);
+            metricsMetadataCachingSource
+                    .addMetadataCacheInvalidationTotalTime(stopWatch.stop().elapsedMillis());
+            metricsMetadataCachingSource.incrementMetadataCacheInvalidationSuccessCount();
```

Comment:
I was thinking to derive it but I guess it will be easier to just have the metric explicitly, so I will add one for failure count. 
I am not sure how total time in case of invalidation failure using the same metric will be helpful? Please let me know.